runtime/lua/vim/lsp.lua

```diff
@@ -283,7 +283,29 @@ end
 ---
 --- Directory markers (.e.g. '.git/') where the LSP server will base its workspaceFolders,
 --- rootUri, and rootPath on initialization. Unused if `root_dir` is provided.
---- @field root_markers? string[]
+---
+--- It's possibile to specify marker's priority. The default behaviour performs a
+--- directory-up first search, but it can be configured to look for multple markers in each
+--- directory before going up.
```

Comment:
We aren't specifying a marker's priority, we're allowing different markers to be ordered by priority. Maybe this is splitting hairs a bit, but when I read "specify a marker's priority" I'm expecting to see some kind of map where each marker can be assigned an arbitrary number, which isn't what this API provides.

```suggestion
--- Each entry in this list is a set of one or more markers. For each set, Nvim
--- will search upwards for each marker contained in the set. If a marker is
--- found, the directory which contains that marker is used as the root
--- directory. If no markers from the set are found, the process is repeated
--- with the next set in the list.
```